content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,113 @@
+description: Istio 1.9 release announcement.
+publishdate: 2021-02-09
+release: 1.9.0
+skip_list: true
+aliases:
+    - /news/announcing-1.9
+    - /news/announcing-1.9.0
+---
+
+We are pleased to announce the release of Istio 1.9! Officially supported versions of Kubernetes releases are 1.17 to 1.20
+for this Istio release.
+
+{{< relnote >}}
+
+This is the first Istio release for 2021.  We would like to thank the entire Istio community, and especially the release
+managers `Shamsher Ansari` (Red Hat), `Steven Landlow` (Google) and `Jacob Delgado` (Aspen Mesh) for helping to get Istio 1.9.0
+published.
+
+The core focus of the 1.9 release of Istio was to improve [Day 2 operations](/docs/reference/glossary/#D). Led by
+feedback from the UX Working Group, most notably by Mitch Connors (Google) and Christian Posta (Solo.io), the
+[Istio TOC](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md) directed the community to
+stabilize the installation and upgrade experiences, as well as to align core features with published
+[Feature Status](https://istio.io/latest/about/feature-stages/) criteria. Thank you to everyone who participated in our
+User Experience survey for helping to make Istio 1.9 be our most successful release to date.
+
+More information will be published in the future detailing the Istio community's plans for 2021 in an upcoming blog.
+
+Here are some highlights for this release:
+
+## Feature promotion
+
+### Virtual machine installation promoted to Beta
+
+With many users needing to expand their service mesh to include Virtual Machines from existing systems, managing the
+complexity of installation is paramount to user success. As such, Virtual Machine support has continued to be a high
+priority for the Istio community. Since Istio 1.6, the community has made major improvements each release for VM
+support. As of Istio 1.9, [Virtual Machine Installation](/docs/setup/install/virtual-machine/) has been promoted to Beta.
+
+All of the supporting documents for this feature have been updated to make it as easy as possible for you to add Virtual
+Machines to your service mesh:
+
+* [Virtual Machine Installation](/docs/setup/install/virtual-machine/) to get started.
+* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
+* [Debugging Virtual Machines](/docs/ops/diagnostic-tools/virtual-machines/) to learn more about troubleshooting issues with virtual machines.
+* [Bookinfo with a Virtual Machine](/docs/examples/virtual-machines/) to learn more about connecting virtual machine workloads to Kubernetes workloads.
+
+VM support will continue to be a high priority for us in 2021, and you can expect us to see more feature
+graduation in upcoming releases.
+
+### External Istiod promoted to Beta
+
+As with VMs, management of external clusters is vital as Mesh Admins increasingly need to expand their service meshes
```

Comment:
I'm confused by this section. Are you talking about external Istiod or multi cluster?

Reply:
I believe a lot of the work done in Istio 1.9 has been for external istiod.

cc @linsun @irisdingbj I know the both of you have worked a lot in this area for Istio 1.9. Can you help with this section? I don't have much experience with the work done related to external istiod.

Reply:
See my comment earlier, i think we need to reach beta checklist first.

Reply:
Updated and reached out to nate and costin. Resolving this in lieu of the other thread.